.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: xenial
 
 branches:
   only:
-    - "/^(feature|bugfix)\\/SCC-[0-9]+.*$/"
+    - "/^(feature|bugfix)\\/SCAT-[0-9]+.*$/"
```

Comment:
Did we want to keep both? In case of infra stuff that we might do off an `SCC-##`?

Reply:
Nice one - updated now to work from both (I think!)